src/sentry/rules/processing/delayed_processing.py

```diff
@@ -539,6 +545,12 @@ def fire_rules(
                         "rule_id": rule.id,
                     },
                 )
+    if features.has("organizations:workflow-engine-process-workflows", project.organization):
+        metrics.incr(
+            "post_process.process_rules.fired_rules",
+            amount=triggered_rules,
+            tags={"event_type": 1},
```

Comment:
Is `event_type` of 1 some hard coded thing for you to identify or is it from an enum?

Reply:
oh nvm I see it's `DEFAULT_TYPE_ID`, could you use that instead?